config/checkstyle_non_main_files_checks.xml

```diff
@@ -17,9 +17,22 @@
     <!-- Miscellaneous -->
     <module name="NewlineAtEndOfFile" />
     <module name="RegexpSingleline">
+      <property name="id" value="lineLength"/>
```

Comment:
please move this update and all related to this changes in other files inn separate commit of the same PR "minor: make unique id 'lineLength' for RegexpSingleline to avoid conflict with future instances the same Check"